### PR TITLE
session has expired page

### PR DESCRIPTION
**Pull request by kyhampton**
Designed and styled session expiration page.

### CODE REVIEW

File: LMS/Pages/SessionExpired.razor [hunk 1]
```diff
@@ -2,11 +2,19 @@
...
+        or you can go there <a id="expired-home" href="/">now.</a></p>
+    </div>
+</div>
 
-@code {
-    protected async override Task OnInitializedAsync()
-    {
-        await Task.Delay(2000).ContinueWith(t => NavMan.NavigateTo("/"));
+    @code {
+        //protected async override Task OnInitializedAsync()
```
Comment (srosy) [inline LMS/Pages/SessionExpired.razor:16]: Double-check your work before you make a pull request. This should be uncommented.

**Review by srosy — CHANGES_REQUESTED** [review]
Please uncomment the timeout code in the sessionexpired view. See comment(s).

File: LMS/Pages/SessionExpired.razor (hunk 1 above)
Comment (kyhampton) [reply LMS/Pages/SessionExpired.razor]: Thanks for finding that. Uncommented.

**Review by srosy — APPROVED** [review]
Good job!

**Pull request merged**